libraries/DataFlash/LogStructure.h

```diff
@@ -1039,6 +1041,7 @@ enum LogMessages {
     LOG_PIDY_MSG,
     LOG_PIDA_MSG,
     LOG_PIDS_MSG,
+    LOG_PIDL_MSG,
```

Comment:
To the best of my knowledge it doesn't matter, if someone else knows differently then please let me know! As far as I know this is simply used for coordinating what message prefix is associated with message, and since a log is self describing on format it doesn't matter.